angularjs-portal-home/src/main/webapp/my-app/search/controllers.js

```diff
@@ -61,6 +62,7 @@ define(['angular', 'portal/search/controllers', 'my-app/marketplace/controllers'
       var initWiscDirectorySearch = function(){
         wiscDirectorySearchService.wiscDirectorySearch($scope.searchTerm).then(
           function(results){
+            $scope.directorySearchLoading = false;
```

Comment:
Make sure to set this to false in the error block also. If it fails, then it would load forever (in its current state)